app/models/color_scheme.rb

```diff
@@ -343,8 +345,9 @@ def self.hex_cache
   after_save_commit :publish_discourse_stylesheet, unless: :skip_publish
   after_save_commit :dump_caches
   after_destroy :dump_caches
+  after_destroy :destroy_remote_original
```

Comment:
Do we also need a validation to prevent `remote_copy` schemes from being updated accidentally?

Reply:
We do need to allow edits to remote copies because when a theme update changes colors of a theme-owned palette, we make those changes to the remote copy.